test/e2e/storage/persistent_volumes-local.go

```diff
@@ -658,7 +658,7 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 			var (
 				pvc   *v1.PersistentVolumeClaim
 				pods  = map[string]*v1.Pod{}
-				count = 50
+				count = 2
```

Comment:
The 50 here was to stress the scheduler and try to uncover races, although an e2e test may not be the right place to do it. @cofyc can the integration stress tests exercise the same race?

Reply:
yes, it's better to stress the scheduler in integration tests. we can convert this test to integration stress test